src/Symfony/Component/HttpFoundation/Tests/HeaderTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpFoundation\Tests;
+
+use PHPUnit\Framework\TestCase;
+
+class HeaderTest extends TestCase
```

Comment:
I'd suggest to name this RequestFunctionalTest instead

Reply:
Done, thanks.